### PR TITLE
Improved celery extension signals

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: django_celery_extensions/decorator.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from django.db.transaction import Atomic
+
+
+tmp.__exit__ = Atomic.__exit__
```
Comment (rubickcz) [inline django_celery_extensions/decorator.py]: tmp is not defined

**Review by rubickcz — APPROVED** [review]

File: django_celery_extensions/decorator.py (hunk 1 above)
Comment (matllubos) [reply django_celery_extensions/decorator.py]: yes this is some error

Comment (Formulka) [reply django_celery_extensions/decorator.py]: what is this even used for?

Comment (matllubos) [reply django_celery_extensions/decorator.py]: I must delete this file

**Pull request merged**